src/ViewModels/src/Output/OutputViewModel.cs

```diff
@@ -21,15 +34,49 @@ public string OutputContent
                 RaisePropertyChangedEvent("OutputContent");
             }
         }
-        
+
         public Process ActiveProcess { get; set; }
 
+        public bool OutputIsAppLogs
+        {
+            get => _outputIsAppLogs;
+            set
+            {
+                _outputIsAppLogs = value;
+                RaisePropertyChangedEvent("OutputIsAppLogs");
+            }
+        }
+
+        public bool OutputPaused
+        {
+            get => _outputPaused;
+            set
+            {
+                _outputPaused = value;
+                RaisePropertyChangedEvent("OutputPaused");
+            }
+        }
+
         public void AppendLine(string newContent)
         {
-            OutputContent += $"{newContent}\n";
+            if (!OutputPaused && !newContent.StartsWith("Retrieving logs for app"))
```

Comment:
"not bool" and "not bool"

 can be rewritten as
 
 "not (bool or bool)"

There fewer the number of nots, the easier it is for the human brain to process

E.g, this could be written:
`if (!(OutputPaused || newContent.StartsWith("Retrieving logs for app")))`

Reply:
The former is honestly easier for my eyes to parse personally (idk why... maybe the added parens in the latter trip me up?) but I see where you're coming from :)